streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -939,6 +939,11 @@ private StreamThread createAndAddStreamThread(final long cacheSizePerThread, fin
         return streamThread;
     }
 
+    //For KafkaStreamsTest
```

Comment:
Why? Could you please add more details?

Reply:
I have update code, thanks for all your comments.